conans/test/integration/cache/short_paths_test.py

```diff
@@ -146,3 +146,10 @@ def test_info_paths(self):
         client.run("export . test/1.0@")
         client.run("info test/1.0@ --paths")
         client.run("info test/1.0@ --paths")
+
+    def test_config_file_user_short_path_none(self):
```

Comment:
Duplicated test? What is this testing?

Reply:
Not duplicated, we didn't test when passing None to user_home_short in conan.conf. We already test using env var, but it's not enough, the current bug exists because we only tested the env var.

According https://docs.conan.io/en/latest/reference/env_vars.html#conan-user-home-short, when None, Conan should use the regular home folder for package caching.

Reply:
I mean, the contents of the test are identical to the above ``test_info_paths``. What am I missing?

Reply:
Ops! Now I see your point, it should not be here. I'll remove this change.